generators/client/templates/angular/src/main/webapp/app/layouts/index.ts.ejs

```diff
@@ -28,3 +28,41 @@ export * from './navbar/active-menu.directive';
 export * from './profiles/page-ribbon.component';
 export * from './profiles/profile.service';
 export * from './profiles/profile-info.model';
+// Imports all fontawesome core and solid icons
+
+import fontawesome from '@fortawesome/fontawesome';
```

Comment:
I'm not sure layouts/index.ts is the best place for these global imports, personally I would have put them in vendor.ts as we do for CSS but I could be wrong.

Reply:
That's exactly why I wanted to submit the PR soon, so I could get some input of the preferred place for the global imports. I'll move it vendor.ts. 👍